lumpy-llamas/frontend/js/message-details.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div v-if="ready">
+    <router-link :to="{ name: 'list-messages'}"><h2>Back to message list</h2></router-link>
+    <h3 class="some-heading">{{myStuff[0].subject}} </h3>
+    <div class="terminal-timeline">
+    <div class="terminal-card" v-for="item in myStuff">
+      <header id="card-header">{{item.from_user_id}} on {{item.created_date | moment("DD/MM/YY/HH:MM")}}</header>
+      <div>
+        {{item.message}}
+      </div>
+    </div>
+    </div>
+       <textarea v-model="message" id="taera" cols="30" rows="5" name="tarea" placeholder="Enter your message" minlength="3"></textarea>
```

Comment:
Can you instead use the global prompt? The idea is that this should mimic a terminal bbs, which means everything done in a single prompt.